### DISCUSSION TITLE
Bioluminescent

### DISCUSSION THREAD

**Discussion by KeenPhD** [carlssonemil/interstellar#40] [Ideas]
I may simply be missing where it is, but will all of the zombies camos for MW2 guns (bioluminescent mastery camo) be added to this MW3 tracker or to the orion tracker?
Thank you! This is the best camo tracker out there!

**Comment by carlssonemil** [comment]
@KeenPhD, yes! I'm planning on adding a zombies section to the Orion tracker as well! I've just been swamped with work, life and general tweaking with the Interstellar tracker! 😁 But I have it on my todo-list and will try to get it done as soon as I can!

**Reply by KeenPhD** [reply]
Of course I understand that! I mostly just wasn't sure if I was simply missing it. Thanks for making such a great tool!
